src/Extensions/ServiceFabricGuest.Abstractions/options/ServiceFabricRestClientOptions.cs

```diff
@@ -1,19 +1,54 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
 using System.ComponentModel.DataAnnotations;
-
+using System.Linq;
+using Microsoft.Omex.Extensions.Abstractions;

```

Comment:
nit: sort

Reply:
I think this one is OK as we usually keep System using directives first